Assets/_Application/Layers/LayerTypes/HierarchicalObject/HierarchicalObjectLayerGameObject.cs

```diff
@@ -58,6 +61,7 @@ private BoundingBox CalculateWorldBoundsFromRenderers()
 
         protected virtual void Awake()
         {
+            snappingCullingMask = (1 << LayerMask.NameToLayer("Terrain")) | (1 << LayerMask.NameToLayer("Buildings"));
```

Comment:
Kan dit ook niet beter een serializable reference zijn die aan te passen is in de prefab waar die bij hoort?

Reply:
Ah, het is de bedoeling dat het harcoded is - ik dacht dat het makkelijk zou zijn om instelbaar te hebben; maar dan komt dit overeen met het DSM - Digital Surface Model